sound/soc/sof/topology.c

```diff
@@ -1886,15 +1886,28 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 			  struct snd_soc_dapm_route *route)
```

Comment:
/data/pbossart/ktest/sof-dev/sound/soc/sof/topology.c: In function ‘sof_widget_ready’:
/data/pbossart/ktest/sof-dev/sound/soc/sof/topology.c:1421:5: warning: ‘ret’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  if (ret < 0 || reply.rhdr.error < 0) {